### PR TITLE
[web] Remove the benchmarks of the HTML renderer

### PR DESCRIPTION
**Pull request by mdebbar**
As the title says, this PR deletes the HTML renderer's benchmarks.

### CODE REVIEW

File: dev/benchmarks/macrobenchmarks/lib/src/web/bench_text_layout.dart [hunk 1]
```diff
@@ -225,13 +198,16 @@ class BenchBuildColorsGrid extends WidgetBuildRecorder {
...
-        reported: true,
-      );
-    }
+    // if (showWidget) {
+    //   profile!.addDataPoint(
+    //     'text_layout',
+    //     Duration(microseconds: _textLayoutMicros.toInt()),
+    //     reported: true,
+    //   );
+    // }
```
Comment (yjbanov) [inline dev/benchmarks/macrobenchmarks/lib/src/web/bench_text_layout.dart:207]: Do we need this?

**Review by yjbanov — APPROVED** [review]

File: .ci.yaml [hunk 2]
```diff
@@ -1502,7 +1502,7 @@ targets:
 
   - name: Linux web_benchmarks_canvaskit
     recipe: devicelab/devicelab_drone
-    presubmit: false
+    presubmit: true
```
Comment (ditman) [inline .ci.yaml:1505]: Do we want to flip these to presubmit permanently?

Comment (mdebbar) [reply .ci.yaml:1505]: Nope, I'm enabling them temporarily to check that benchmarks are still running and I didn't break something.

**Review by ditman — APPROVED** [review]
LGTM!

**Pull request merged**